ProcessList.h

```diff
@@ -81,12 +81,15 @@ typedef struct ProcessList_ {
    memory_t usedSwap;
    memory_t cachedSwap;
 
-   unsigned int cpuCount;
+   unsigned int activeCPUs;
+   unsigned int existingCPUs;
```

Comment:
No big deal but I find the naming slightly unclear - AIUI 'active' means 'online' and 'existing' means the CPU exists but is either online or offline?  An alternative might be to use 'onlineCPUs' and something like 'actualCPUs' or 'realCPUs' here?

Reply:
I'd prefer `actualCPUs` over `realCPUs` to not clash with virtualization stuff …